portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,31 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> comments = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    if ( comments.size() == 0){
+      comments.add("Radiohead is the best ever band");
+      comments.add("I dont really agree with you, Bary. We should talk more about this.");
+      comments.add("There is nothing to talk about, they are great and this is it.");
+    }
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Teo!</h1>");
+    response.getWriter().println(convertToJSON(comments));
+  }
+  
+  private String convertToJSON(ArrayList<String> commentsArray){
```

Comment:
This function doesn't access the object data, should be static? WDYT?

Reply:
I was thinking to create a class Comment (String userName, String message, Date date), and one CommentsSection(ListArray<Comment> comments).  This way I could have methods within the class. But until then surely it is best for that method to be static.